app/core/settings.py

```diff
@@ -163,3 +163,10 @@
 DIGEST_CONTENT_TYPE = 'application/vnd.elife.digest+json;version=1'
 DIGESTS_CONTENT_TYPE = 'application/vnd.elife.digest-list+json;version=1'
 ERROR_CONTENT_TYPE = 'application/problem+json'
+
+ELIFE_BUS = {
+    'env': ENVIRONMENT,
+    'name': os.environ.get('BUS_NAME'),
+    'region': os.environ.get('AWS_DEFAULT_REGION'),
```

Comment:
ok for naming `AWS_DEFAULT_REGION` so as SDKs like boto accept it, but `subscriber` is our term so it should be `BUS_SUBSCRIBER`